qwik-graphql-tailwind/src/components/repo-header/index.tsx

```diff
@@ -3,29 +3,41 @@ import { useLocation } from '@builder.io/qwik-city';
 import { RepoActionButtons } from '../repo-action-buttons';
 import { RepoHeading } from '../repo-heading';
 import TabNavigation from '../tab-navigation/tab-navigation';
-import { tabList } from './tab-lists';
+import { createTabList } from './tab-lists';
 
 interface RepoHeaderProps {
   name: string;
   owner: string;
   stargazerCount: number | string;
   forkCount: number | string;
   watcherCount: number | string;
+  issuesCount: number;
+  prCount: number;
 }
 
-export const RepoHeader = component$(({ name, owner, watcherCount, stargazerCount, forkCount }: RepoHeaderProps) => {
-  const { pathname } = useLocation();
-  const basePath = `${owner}/${name}`;
+export const RepoHeader = component$(
+  ({ name, owner, watcherCount, stargazerCount, forkCount, issuesCount, prCount }: RepoHeaderProps) => {
+    const { pathname } = useLocation();
+    const basePath = `${owner}/${name}`;
 
-  return (
-    <div className="pt-6 px-12 bg-gray-100 border-b border-gray-300">
-      <div className="flex justify-between items-center">
-        <RepoHeading name={name} owner={owner} />
-        <RepoActionButtons watcherCount={watcherCount} stargazerCount={stargazerCount} forkCount={forkCount} />
+    return (
+      <div className="pt-6 px-12 bg-gray-100 border-b border-gray-300">
+        <div className="flex justify-between items-center">
+          <RepoHeading name={name} owner={owner} />
+          <RepoActionButtons watcherCount={watcherCount} stargazerCount={stargazerCount} forkCount={forkCount} />
+        </div>
+        <div className="mt-6">
+          <TabNavigation
+            tabs={createTabList({
+              issueCount: issuesCount,
+              pullRequestCount: prCount,
```

Comment:
can we assign the createTabList to a variable and use the variable here instead?